cmd/tailscale/cli/speedtest.go

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
```

Comment:
Ah. I had meant to put this in tailscale/cmd/speedtest, as a separate command that we could work with, and refactor it to run the server within tailscaled as a next step.

The trouble with putting it into tailscale/cli is that it would become visible to users right away in the next release, and I think we don't want that until the server is running automatically in tailscaled.

github.com/peterbourgon/ff/v2/ffcli still works as a standalone tailscale/cmd/speedtest, so I don't think this would be difficult to move there.

Reply:
Moved it out of the Tailscale cli.